targets/TARGET_NUVOTON/TARGET_M480/crypto/crypto-misc.cpp

```diff
@@ -19,15 +19,16 @@
 #include "mbed_assert.h"
 #include "mbed_critical.h"
 #include "mbed_error.h"
+#if defined(MBED_CONF_RTOS_PRESENT) && MBED_CONF_RTOS_PRESENT
```

Comment:
We shall just use `#if MBED_CONF_RTOS_PRESENT`, we consolidated recently drivers to use this as well (no toolchain should warn if not defined, should be set to 0).